src/LCM2ROSControl.hpp

```diff
@@ -19,19 +19,23 @@
 #include "lcmtypes/bot_core/ins_t.hpp"
 #include "lcmtypes/bot_core/joint_angles_t.hpp"
 #include "lcmtypes/bot_core/atlas_command_t.hpp"
+#include "lcmtypes/drc/behavior_transition_t.hpp"
 
 #include <set>
 #include <string>
 #include <vector>
+#include <chrono>
 
 namespace valkyrie_translator
 {
 
-   typedef struct _joint_command {
-    double position;
-    double velocity;
-    double effort;
+  enum Behavior {
+    FREEZE,
+    POSITION_CONTROL,
+    NORMAL
+  };
```

Comment:
Minor thing, but I would give preference to enum classes now that they're available, as they're strongly typed (can't pass in an int for a Behavior by accident).